server/pages/index.js

```diff
@@ -92,15 +94,14 @@ function generateStaticUrls( request ) {
 		urls[ file.path ] = getUrl( file.path, file.hash );
 	} );
 
-	const assets = request.app.get( 'assets' );
+	// in production, only load assets file the first go around
+	if ( ! assets || process.env.NODE_ENV === 'development' ) {
+		assets = JSON.parse(
+			fs.readFileSync( path.join( __dirname, '../', 'bundler', 'assets.json' ), 'utf8' )
+		);
+	}
 
-	assets.forEach( function( asset ) {
-		const name = asset.name;
-		urls[ name ] = asset.url;
-		if ( config( 'env' ) !== 'development' ) {
-			urls[ name + '-min' ] = asset.url.replace( '.js', '.min.js' );
-		}
-	} );
+	forEach( assets, ( asset, name ) => urls[ name ] = asset.js );
```

Comment:
i'll admit it took me a minute to parse out that `asset.js` was `asset[ 'js' ]` and not a straight filename :D